src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -349,18 +352,8 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
 
     @computed
     get chart() {
-        const {
-            BAR_CHART,
-            SURVIVAL,
-            CNA_GENES_TABLE,
-            TABLE,
-            SCATTER,
-            PIE_CHART,
-            MUTATED_GENES_TABLE,
-            FUSION_GENES_TABLE,
-        } = ChartTypeEnum;
         switch (this.chartType) {
-            case PIE_CHART: {
+            case ChartTypeEnum.PIE_CHART: {
```

Comment:
this switch statement is getting pretty huge.   we should instead have a map.   
```
const chartMap = {};
chartMap[enum.PIE_CHART] = ()=><PieChart>
```
then the chart method can just look it up.

Reply:
let's do the refactoring in another pr